File: apk_telepon_sms/app/src/main/java/com/fikri/apple/apk_telepon_sms/MainActivity.java
package com.fikri.apple.apk_telepon_sms;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    Button teleponn, smss;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        teleponn = findViewById(R.id.telpon);
        smss = findViewById(R.id.sms);

        teleponn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent telponnn = new Intent(MainActivity.this, telpon.class);
                startActivity(telponnn);
            }
        });

        smss.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent smss = new Intent(MainActivity.this, sms.class);
                startActivity(smss);

            }
        });


    }
}

File: UIKurirApp/app/src/main/java/com/fikri/apple/uikurirapp/main1/MainActivity.kt
package com.fikri.apple.uikurirapp.main1

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.LinearLayoutManager
import com.fikri.apple.uikurirapp.R
import com.fikri.apple.uikurirapp.main2.Main2Activity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    private val listMain = ArrayList<MainModel>()
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        listMain.addAll(getListMain())
        initRecylerView()

        btnAntar.setOnClickListener {
            startActivity(Intent(this,Main2Activity::class.java))
        }
    }

    private fun initRecylerView() {
        rvMain.setHasFixedSize(true)
        rvMain.layoutManager = LinearLayoutManager(this)
        val listMainAdapter = ItemMainAdapter(listMain)
        rvMain.adapter = listMainAdapter
    }

    private fun getListMain(): ArrayList<MainModel> {
        val dataNum = resources.getStringArray(R.array.array_number)
        val dataName = resources.getStringArray(R.array.array_name)
        val dataAddress = resources.getStringArray(R.array.array_address)
        val dataReceipt = resources.getStringArray(R.array.array_receipt)
        val dataImage = resources.getStringArray(R.array.array_image)

        val list = ArrayList<MainModel>()
        for (position in dataNum.indices){
            val model = MainModel(
                dataNum[position],
                dataName[position],
                dataReceipt[position],
                dataAddress[position],
                dataImage[position]
            )

            list.add(model)
        }
        return list
    }
}
